### PR TITLE
Feature/incident tests

### PR DESCRIPTION
**Pull request by joseph-witzke**
# Description
Made edits to the "getIncidentById" model function test. Tests pass, but needs further review. 

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [√] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status
- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [√ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?
- [ ] Test A
- [ ] Test B

# Checklist
- [√ ] My code follows the style guidelines of this project
- [√ ] I have performed a self-review of my own code
- [√ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [√ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [√] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

Screen Shot 2021-08-10 at 8 11 35 PM

### CODE REVIEW

**Comment by dylan-rinella** [comment]
Looks great overall, this helped me get unstuck on a couple of unit tests that Ive bee working on.

File: data/seeds/001-incidents.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 exports.seed = function (knex) {
   // Deletes ALL existing entries
-  return knex('incidents')
+  return knex('force_ranks')
```
Comment (devbrunopaula) [inline data/seeds/001-incidents.js:3]: Great catch here. There isn't any incidents table.

**Review by devbrunopaula — APPROVED** [review]
Code looks clean, and its following industry standards.

**Review by RyderBlues — APPROVED** [review]
Good to go!

**Review by RyderBlues — CHANGES_REQUESTED** [review]
Package-lock merge issues

**Pull request closed without merging**